app/index.html

```diff
@@ -9,10 +9,10 @@
   <title>My AngularJS App</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/normalize.css">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/main.css">
+  <link rel="stylesheet" href="lib/html5-boilerplate/dist/css/normalize.css">
```

Comment:
I would rather not introduce a task runner (grunt, gulp etc). Everyone will have a different preference and this seed project has been tranditionally quite unopinionated. Also, keep n mind that `angular-seed` is used as a base for `angular-phonecat`, so simplicity is a feature :smiley:

How about using an npm script (e.g. postinstall) that copies the necessary files from `node_modules/...` to `app/lib/`? It is still less straightforward than `bower install` imo, but more straightforward than using a task runner or a rewriting server. WDYT?